sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/SonarLintProfileExporter.java

```diff
@@ -19,13 +19,14 @@
  */
 package org.sonar.plugins.csharp;
 
+import org.sonar.api.rules.RuleFinder;
```

Comment:
It looks like this API has been deprecated since 5.1 : http://javadocs.sonarsource.org/5.3/apidocs/org/sonar/api/rules/RuleFinder.html

Do we have a long-term plan to replace this with a supported API, or a ticket to track that further work is required?